config/peerpods/podvm/aws-podvm-image-cm.yaml

```diff
@@ -18,12 +18,12 @@ data:
   # Pod VM sources
   # If changing the source, then ensure the respective payload binaries are available
   # for the new source
-  CAA_SRC: "https://gitlab.cee.redhat.com/osc/midstream/cloud-api-adaptor"
-  CAA_REF: "v0.8.0-1.6"
+  #CAA_SRC: "" ## set to cloud-api-adaptor sources URL for custom podvm builds
+  #CAA_REF: "" ## set to tag/branch for the cloud-api-adaptor code to use
 
   # Booleans
   INSTALL_PACKAGES: "no"
-  DOWNLOAD_SOURCES: "no"
+  DOWNLOAD_SOURCES: "no" ## set to "yes" for custom podvm builds. Need to set CAA_SRC and CAA_REF accordingly
```

Comment:
Hi @littlejawa ! I think we should hide `DOWNLOAD_SOURCES` from final users too.

Reply:
Yes, IMO it could even be fully removed. In order to do `git clone` we just need `CAA_SRC`. If some other ref than the default branch is wanted then `CAA_REF` should be provided. No need to do any validation in this case.

Reply:
My suggestion will be to just document the variables (and not remove anything right now), and sync with end-user documentation to ensure we explicitly state out config options that should not be touched.
Given we are looking at bootc based image build, things will change soon. So may be we should focus the effort there.